teknologkoren_se/templates/admin/file.html

```diff
@@ -0,0 +1,34 @@
+{% extends "admin/base.html" %}
+
+{% set title = "Redigera inlägg" %}
+
+{% from "macros.html" import form_entry, form_entry_row %}
+
+{% block body %}
+
+<h1>Fil</h1>
+{% if file %}
+<code>{{ file.filename }}</code>
+
+<p>URL till filen:
+  <a href="{{ url_for_file(file.filename) }}">{{ url_for_file(file.filename) }}</a>
+</p>
+{% endif %}
+
+{% if file %}
+<h2>Ersätt fil</h2>
+
+<p>Om du laddar upp en ny fil kommer den nuvarande bytas ut på alla platser den redan används. Den tidigare filen kommer inte gå att nå utan att ha URL:en eller att fråga webmaster.</p>
```

Comment:
Betyder detta alltså att URLen ändå ändras när man byter ut en fil?

Jag trodde att detta skulle vara en lösning för när vi vill skriva ut QR-koder till ett programblad innan det är klart (eller kunna fixa typos i programbladet efter utskrift av QR-koden). Men man måste alltså även fortsatt gå in manuellt på servern och ersätta filen i dessa fall?

Reply:
Hmm, ja filnamnet ändras och man kan inte ladda upp en fil med ett namn som redan finns. Man skulle kunna lägga till en checkbox för "behåll filnamn" som tar bort eller flyttar den gamla filen och byter namn på den nya så att urlen är densamma.